client/client-multi/client-c/toolchains/cc32xx.cmake

```diff
@@ -16,21 +16,78 @@
 
 include(CMakeForceCompiler)
 
-if(DEFINED WIN32)
-  set(EXECUTABLE_SUFFIX .exe)
-else()
-  set(EXECUTABLE_SUFFIX)
-endif()
-
-if(NOT DEFINED KAA_TOOLCHAIN_PATH)
-  set(KAA_ARM_TOOLCHAIN "/opt/kaa/gcc-arm-none-eabi")
+set(CMAKE_SYSTEM_NAME Generic)
+
+set(ARM_GCC_COMPILER "arm-none-eabi-gcc${CMAKE_EXECUTABLE_SUFFIX}")
+
+# Find toolchain path
+
+if (NOT DEFINED ENV{CC32XX_TOOLCHAIN_PATH})
+  # Check if GCC is reachable.
+  find_path(TOOLCHAIN_PATH bin/${ARM_GCC_COMPILER})
+
+  if (NOT TOOLCHAIN_PATH)
```

Comment:
This will break original build behavior: if toolchain path wasn't set, it sets /opt/kaa/gcc-arm-none-eabi as toolchain path.